### ISSUE TITLE
Menu component build failed due to AccessibilityInfo on macOS

### ISSUE THREAD

**Issue by likeSo** [GeekyAnts/NativeBase#4421]
**Describe the bug**
I am enabling macOS support on my project.
Once i import `Menu` component from native-base, it will not build and show this error:
![image](https://user-images.githubusercontent.com/12247340/146491509-3bd71052-4304-4239-86e6-3862ca53377f.png)

**To Reproduce**
Steps to reproduce the behaviour:
1. Run `npx react-native-macos-init` in a bare React Native project.
2. `import {Menu} from 'native-base'
3. `npx react-native run-macos`
4. You got the error

**Expected behaviour**

**CodeSandBox/Snack link**
It would be easier for us to debug if you can reproduce the error in a codesandbox link or a snack link. You can use the [codesandbox template](https://codesandbox.io/s/native-base-v3-template-mq6ox)/[snack template](https://snack.expo.io/@mdrehman/nativebase-template).

**Screenshots**

**Platform**

- NativeBase version [e.g. 3.0.1]: 3.2.1
- Device: [e.g. iPhone6]: My Mac
- OS: [e.g. iOS8.1]: 11.6
- Browser version [e.g. 22]
- React Native version [e.g. 0.64]: 0.64.3
- Expo sdk version [e.g. 41]: 43.0.0
- node [e.g. 14.16.1]: 14.17.5
- yarn or npm [e.g. 6.1]: yarn 1.22.15

**Additional information**

1. Add any other context about the problem here.
2. Is the bug present in both iOS and Android or in any one of them?
3. If you want your issue to be looked at quicker, please attach a snack reproducible with your issue. Makes it easier for us!

**Comment by mayank-96** [comment]
@likeSo Thanks for reporting the issue, we'll look into this.

**Comment by likeSo** [comment]
Don't close plz. I am still waiting for macOS support for this library

**Comment by likeSo** [comment]
Any update on this?

**Comment by surajahmed** [comment]
We're not currently focusing on macOS support. We might add this to our roadmap in the future. Thanks for your patience. Closing this for now. Feel free to reopen if you have something to discuss regarding this.
